pkg/lib/bundle/generate.go

```diff
@@ -0,0 +1,267 @@
+package bundle
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	defaultPermission   = 0644
+	registryV1Type      = "registry+v1"
+	plainType           = "plain"
+	helmType            = "helm"
+	annotationsFile     = "annotations.yaml"
+	dockerFile          = "Dockerfile"
+	manifestsDir        = "manifests/"
```

Comment:
Can we make this consts here and below upper case so they can be used in other packages?